core/cog-view.c

```diff
@@ -396,3 +442,22 @@ cog_view_get_use_key_bindings(CogView *self)
     CogViewPrivate *priv = cog_view_get_instance_private(COG_VIEW(self));
     return priv->use_key_bindings;
 }
+
+/**
+ * cog_view_get_viewport: (get-property viewport)
+ * @self: A view.
+ *
+ * Gets viewport where the view is attached.
+ *
+ * Returns: The viewport what the view belongs.
```

Comment:
This is missnig missing `(transfer none)`  and `(nullable)` annotations:

```suggestion
 * Returns: (transfer none) (nullable): The viewport what the view belongs.
```

That is because a new ref is not added to the returned object, and the returned value can be `NULL` while the view has not been added to any viewport.